### ISSUE TITLE
Convergence during training

### ISSUE THREAD

**Issue by 1962975362** [eriklindernoren/PyTorch-YOLOv3#489]
The overall loss of the model was decreasing, but the accuracy of the three yolo sub-models was jumping, the sense of unconvergence, and the loss varied between 2 and 4

some result：
Epoch----
---- [Epoch 193/200, Batch 229/230] ----
+------------+----------------+----------------+----------------+
| Metrics    | YOLO Layer {0} | YOLO Layer {1} | YOLO Layer {2} |
+------------+----------------+----------------+----------------+
| grid_size  | 10             | 20             | 40             |
| loss       | 1.970637       | 0.978272       | 1.008966       |
| x          | 0.030162       | 0.036681       | 0.073310       |
| y          | 0.022680       | 0.045378       | 0.056900       |
| w          | 0.016418       | 0.016895       | 0.015628       |
| h          | 0.021360       | 0.024979       | 0.020170       |
| conf       | 1.746027       | 0.668247       | 0.663098       |
| cls        | 0.133989       | 0.186092       | 0.179861       |
| cls_acc    | 72.73%         | 54.55%         | 54.55%         |
| recall50   | 0.454545       | 0.545455       | 0.454545       |
| recall75   | 0.454545       | 0.545455       | 0.363636       |
| precision  | 0.500000       | 0.315789       | 0.119048       |
| conf_obj   | 0.508562       | 0.826029       | 0.820714       |
| conf_noobj | 0.004301       | 0.002767       | 0.002256       |
+------------+----------------+----------------+----------------+
Total loss:---3.9578757286071777
---- ETA------0:00:00
Epoch----
---- [Epoch 194/200, Batch 229/230] ----
+------------+----------------+----------------+----------------+
| Metrics    | YOLO Layer {0} | YOLO Layer {1} | YOLO Layer {2} |
+------------+----------------+----------------+----------------+
| grid_size  | 16             | 32             | 64             |
| loss       | 0.690056       | 0.648371       | 0.701728       |
| x          | 0.030520       | 0.028651       | 0.053038       |
| y          | 0.016577       | 0.028671       | 0.100284       |
| w          | 0.010915       | 0.011263       | 0.014480       |
| h          | 0.014761       | 0.019474       | 0.025565       |
| conf       | 0.503859       | 0.441864       | 0.351783       |
| cls        | 0.113425       | 0.118449       | 0.156579       |
| cls_acc    | 75.00%         | 62.50%         | 62.50%         |
| recall50   | 0.750000       | 0.625000       | 0.625000       |
| recall75   | 0.750000       | 0.500000       | 0.500000       |
| precision  | 0.461538       | 0.250000       | 0.071429       |
| conf_obj   | 0.899210       | 0.868416       | 0.946538       |
| conf_noobj | 0.002897       | 0.001578       | 0.001634       |
+------------+----------------+----------------+----------------+
Total loss:---2.0401551723480225
---- ETA------0:00:00
Epoch----

**Comment by ardeal** [comment]
we are experiencing similar issue.
See my issue  #504

**Comment by Flova** [comment]
Duplicate of #504
